app/models/manageiq/providers/embedded_ansible/automation_manager/configuration_script.rb

```diff
@@ -41,8 +41,8 @@ def run(vars = {})
     credentials = collect_credentials(vars)
 
     kwargs = {:become_enabled => vars[:become_enabled]}
-    kwargs[:timeout]   = vars[:execution_ttl].to_i.minutes if vars[:execution_ttl].present?
-    kwargs[:verbosity] = vars[:verbosity].to_i             if vars[:verbosity].present?
+    kwargs[:timeout]   = vars[:execution_ttl].present? ? vars[:execution_ttl].to_i.minutes : 100.minutes
```

Comment:
ok, as discussed, 

- Let's just add a comment/TODO to describe why we have 100.minutes here, and that we need to move this down deeper into automate as a followup 
- We need to make the same change here [1] for automate playbook method type.  This code seems to be defaulting to 60 minutes, which will run into the same issue.

[1]

Reply:
Does it make sense to make `100.minutes` a constant and have the TODO comment exist there?